codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandler.java

```diff
@@ -57,10 +59,41 @@
     public enum ServerHandshakeStateEvent {
         /**
          * The Handshake was complete succesful and so the channel was upgraded to websockets
+         *
+         * @deprecated in favor of {@link HandshakeComplete} class,
+         * it provides extra information about the handshake
          */
+        @Deprecated
         HANDSHAKE_COMPLETE
     }
 
+    /**
+     * The Handshake was complete succesful and so the channel was upgraded to websockets
```

Comment:
`was complete succesful` -> `was completed succesfully`

`and so` -> `and`

add a period at the end of the sentence.